### ISSUE TITLE
No module named tensorboard in the last pull and build 0.7

### ISSUE THREAD

**Issue by HamedMP** [tensorflow/tensorflow#1199]
I just pulled the repo and build it from source and when calling tensorboard in the terminal I get following error:
```shell
Traceback (most recent call last):
  File "/Users/tensorflow/bin/tensorboard", line 7, in <module>
    from tensorflow.tensorboard.backend.tensorboard import main
ImportError: No module named tensorboard
```

Env is Mac OS X El Capitan.

**Comment by caisq** [comment]
Do you have more than one installations of TensorFlow on your system? It could that the new tensorboard binary is trying to import from an older version of TensorFlow installation.

**Comment by HamedMP** [comment]
No, I'm using vitualenv and before instilling 0.7, I removed 0.6 manually

**Comment by vrv** [comment]
Can you try creating a new virtualenv and installing from scratch?  I just checked in a fresh install and that module exists.

**Comment by HamedMP** [comment]
I checked it in new virtualenv and results as follow:

- Installing from the built source by me >> got the same error (in the other virtualenv I'm also using my own build)

- Installing from wheel works fine

I build from the source touching nothing, just git pull and following the instruction on the website.

btw, the reason I built it from source was that the `freeze_graph()` python code doesn't run because of lacking `graph_util.convert_variables_to_constants()` method was not in the wheel but in the git. I think you should update the wheel with new build.

**Comment by vrv** [comment]
Hmm, our new wheels should get that function -- not sure how it didn't make it into the 0.7.0 ones.

As for the tensorboard bug, now I'm seeing it when building from HEAD.  Will look into it and make sure it's fixed in 0.7.1

**Comment by HamedMP** [comment]
Thanks, looking forward to it.

Someone [mentioned](http://stackoverflow.com/a/35295754/1973820) that there is mismatches in the BUILD but I couldn't get it to work by the proposed solution.

**Comment by vrv** [comment]
https://github.com/tensorflow/tensorflow/commit/716b7713e4c8b2d8f093f639ca41816cf4e1c696

I think that should fix it -- I'm cherry-picking that into 0.7.1 right now.  Feel free to let me know if it's still not working.

**Comment by HamedMP** [comment]
Yes, Thank you very much, it solved the issue.
